modules/core/04-channel/v2/types/msgs.go

```diff
@@ -13,6 +13,8 @@ import (
 	ibcerrors "github.com/cosmos/ibc-go/v9/modules/core/errors"
 )
 
+const MaxTimeout uint64 = 86400 // 24h in seconds
```

Comment:
Should we call this `MaxTimeoutDelta` instead? So that it is not interpreted as an absolute value?

Also might be better to be explicit:

```suggestion
const MaxTimeoutDelta uint64 = 24 * 60 * 60 // 24h in seconds
```